pylintrc

```diff
@@ -14,6 +14,7 @@ max-line-length=131
 
 [TYPECHECK]
 ignored-classes=pytest,lxml.etree
+ignored-modules=distutils
```

Comment:
If you look at tox.ini for yelp/yelp_cheetah at version 10 you can see the hack I did for pypy and do a similar thing for py26: https://github.com/Yelp/yelp_cheetah/blob/a72711871db92a6d8cb2a4bea8e2a5adb3262435/tox.ini#L18